content/modules/data-table/versions/1.4.2.um

```diff
@@ -0,0 +1,11 @@
+@version 1.4.2
+  @api Data Table
+    @object hx.dataTable
+      @function objectFeed
+        @param data [Object]
+        @param? options [Object]
+          @property compare [Function]
+            @param cell1 [String]
+            @param cell2 [String]
+            @default
+              @removed
```

Comment:
This gets around the issue with Quantum not allowing us to re-assign a default value
Not sure if there's a better way currently without changing `quantum-api`